app/src/main/java/org/oppia/android/app/onboarding/OnboardingFragment.kt

```diff
@@ -7,13 +7,23 @@ import android.view.View
 import android.view.ViewGroup
 import org.oppia.android.app.fragment.FragmentComponentImpl
 import org.oppia.android.app.fragment.InjectableFragment
+import org.oppia.android.util.platformparameter.EnableOnboardingFlowV2
+import org.oppia.android.util.platformparameter.PlatformParameterValue
 import javax.inject.Inject
+import org.oppia.android.app.onboardingv2.OnboardingFragmentPresenter as OnboardingFragmentPresenterV2
```

Comment:
Perhaps rather than using a separate package, we could ``git mv`` the existing presenter to become ``OnboardingFragmentPresenterV1`` and make the new presenter the ``OnboardingFragmentPresenter``.

If the diffs are hard to see, you could submit a precursor PR that just makes the move and then this one to introduce the new file (which would keep the diffs essentially the same as here).

Reply:
I'm not opposed to this. As to whether there will be a precursor PR, I will report after the refactore attempt.

Reply:
I have decided to do this as a commit to this PR.

Reply:
I think that's fine for this amount of code churn. Larger refactors would benefit from two steps to make the review process cleaner (though the history still gets a bit jumbled post merges on the develop branch; Git doesn't handle move/copies very well unfortunately).